File: Project/src/app/items.ts
export interface Items {
  id: number;
  runes: string;
  images: string;
  itemCode: string;
}

const ELEMENT_DATA: Items[] = [];

export class TableColumnStylingExample {
  displayedColumns: string[] = ['id', 'runes', 'images'];
  dataSource = ELEMENT_DATA;
}

File: src/main/java/pl/grzegorz/projects/WEBDiabloPage/model/entity/Items.java
package pl.grzegorz.projects.WEBDiabloPage.model.entity;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.*;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "items")
public class Items implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(nullable = false, updatable = false)
    private Long id;
    private String runes;
    private String images;
    private String itemCode;
    @JsonIgnore
    public Items(String runes, String images) {
        this.runes = runes;
        this.images = images;
    }
}
